subworkflows/nf-core/align_bowtie2/meta.yml

```diff
@@ -0,0 +1,59 @@
+name: align_bowtie2
+description: Align reads to a reference genome using bowtie2 then sort with samtols
+keywords:
+    - align
+    - fasta
+    - genome
+    - reference
+tools:
+    - bowtie2:
+          description: |
+              Bowtie 2 is an ultrafast and memory-efficient tool for aligning
+              sequencing reads to long reference sequences.
+          homepage: http://bowtie-bio.sourceforge.net/bowtie2/index.shtml
+          documentation: http://bowtie-bio.sourceforge.net/bowtie2/manual.shtml
+          doi: 10.1038/nmeth.1923
+    - samtools:
+        description: |
+            SAMtools is a set of utilities for interacting with and post-processing
+            short DNA sequence read alignments in the SAM, BAM and CRAM formats, written by Heng Li.
+            These files are generated as output by short read aligners like BWA.
+        homepage: http://www.htslib.org/
+        documentation: hhttp://www.htslib.org/doc/samtools.html
+        doi: 10.1093/bioinformatics/btp352
```

Comment:
If we eventually have a website rendering this information, this part could be pulled in from the corresponding modules, avoiding redundant information.

Reply:
I love it! Not sure how to make that magic happen, though. I really wish yaml had native include statements.

Reply:
@ewels is planning to do this (for modules at least): https://nfcore.slack.com/archives/CE7DN1U7M/p1628502439008800

Reply:
Agreed - just need the module identifier. Maybe more metadata for this though: for example repo and git sha? Some discussion on `#tools` about support for non nf-core repos.

I guess it should basically mimic the info saved about modules used in pipelines? And maybe pipelines then only need to record the subworkflows that they use and not the modules used by those subworkflows..?

Reply:
Yeah, I think a `modules` section instead of a `tools` section to not repeat ourselves.

I think we're going to run into issues with testing using specific git sha's for the modules. Sure, end user and the installation can work, but how do we develop and maintain them?

I think like @grst using the specific git sha would be a cool feature. I think we should just make an issue of it and cross that bridge when we've figured out more pain points around this and iterate on it. It's not like this repo hasn't been completely restructured a few times already ;).